doc/classes/DisplayServer.xml

```diff
@@ -1055,7 +1055,7 @@
 				Each [Dictionary] contains two [String] entries:
 				- [code]name[/code] is voice name.
 				- [code]id[/code] is voice identifier.
-				- [code]language[/code] is language code in [code]lang_Variant[/code] format. [code]lang[/code] part is a 2 or 3-letter code based on the ISO-639 standard, in lowercase. And [code]Variant[/code] part is an engine dependent string describing country, region or/and dialect.
+				- [code]language[/code] is language code in [code]lang_Variant[/code] format. The [code]lang[/code] part is a 2 or 3-letter code based on the ISO-639 standard, in lowercase. The [code safe]Variant[/code] part is an engine dependent string describing country, region or/and dialect.
```

Comment:
This seems to imitate the casing of the value itself, which I don't think is very clear in this case. But it matches a similar example in OS.xml, so ultimately I left this one as is.